doc/source/Platforms.rst

```diff
@@ -389,6 +400,10 @@ Remember to activate the ``lua`` module environment and have MacTeX in your sear
 
    spack stack create env --site macos.default [--template jedi-ufs-all] --name jedi-ufs.mymacos
 
+.. note::
+  It is advisable to use the :code:`--template jedi-ufs-all` option on the :code:`spack stack create env` command above.
```

Comment:
Hmmm ... spack-stack is not specific to JCSDA or JEDI. Some people may use it to build UFS_UTILS or GSI or just ESMF or whatever else, and never touch any JEDI code. I suggest to add this type of information to our jedi-docs documentation, similar to what I am trying to collect in the working google doc now.

Once the page on `Environments` with package descriptions, templates etc. are updated, it will hopefully become easier to figure out what users can decide to build. It's easier to start with a small setup and add packages later on if needed, I think.